### DISCUSSION TITLE
Enable Auto Update - 7.4.0

### DISCUSSION THREAD

**Discussion by arvindjshetty** [PowerShell/PowerShell#21220] [Q&A] [answered]
Recently we deployed PowerShell 7.4.0 to over 35K workstations and we are planning to deploy  version 7.4.1 to address the vulnerabilities for the previous version through SCCM. We would like to know if we can enable the "Auto Update" for the MSI package so that we don't need to deploy  newer version of PowerShell whenever its released and the workstations should automatically update to the latest version.

**Comment by kilasuit** [comment, marked as answer]
Yes as long as the settings for Microsoft Update have been set either as part of the install process, or afterwards, then it will update once it is available via Microsoft Update. **However** there is a delay between a new version being released and it being available via Microsoft Update (usually close to 2 weeks) which may mean you may still want to have a tighter control of when an upgrade happens by initiating your own deployment sooner than it would be via Microsoft Update

See this for more detail
https://learn.microsoft.com/en-us/powershell/scripting/install/microsoft-update-faq?view=powershell-7.4#how-soon-after-release-are-updates-advertised-by-microsoft-update
